src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
```

Comment:
Instead of calling each time Color.values() to get Color array, you can create:
`public static Color[] colors = Color.values();`

Since enum is just static fields itself, they won't change at runtime during program, so you can use that to avoid unnecessary method calls and variables

Reply:
Variable 'colors' must be private and have accessor methods @from ide